src/Bicep.Core.UnitTests/Registry/OciModuleRegistryTests.cs

```diff
@@ -784,7 +784,7 @@ private IFeatureProvider GetFeatures(string cacheRootDirectory, bool publishSour
         private Stream CreateSourceStream(string mainBicepContent)
         {
             var mainBicepUri = new Uri("https://hello/main.bicep", UriKind.Absolute);
-            return SourceArchive.PackSourcesIntoStream(mainBicepUri, new ISourceFile[] {
+            return SourceArchive.PackSourcesIntoStream(mainBicepUri, null, new ISourceFile[] {
```

Comment:
Not just novice readers...  And I agree.

I decided it would be better to add an overload without this parameter.

Also, you can just say "and all other uses in all files".  Easier for me to do shift+F12 than for you to do all that typing and us manage it....  Thx.

Reply:
521ad6c41